android_app/app/src/main/java/com/health/openscale/core/database/AppDatabase.java

```diff
@@ -75,5 +75,27 @@ public void migrate(SupportSQLiteDatabase database) {
             }
         }
     };
+
+    public static final Migration MIGRATION_2_3 = new Migration(2, 3) {
+        @Override
+        public void migrate(SupportSQLiteDatabase database) {
+            database.beginTransaction();
+            try {
+                database.execSQL("ALTER TABLE scaleMeasurements ADD COLUMN visceralFat REAL NOT NULL DEFAULT 0");
```

Comment:
It might be that you need to add the columns in the right order so that the final order is the same as when the table is created from scratch.

But a test will tell you if this is needed.